src/components/markdown-component/MarkdownComponent.css

```diff
@@ -0,0 +1,14 @@
+.markdown-basic-component {
+    width: 80%;
```

Comment:
I'm not sure this global styling is great, I think this could be good for styles like font sizes/styles but setting 80% width caused issues when I attempted to implement on the case studies single page. we can discuss next week if you like